### ISSUE TITLE
[MU4 Issue] Fermata on barline not playing

### ISSUE THREAD

**Issue by KaiVinter** [musescore/MuseScore#15926]
**Describe the bug**
Usually, a fermata written on a barline often means to not play for a moment, but when writing fermatas on a barline they don't play nor make any change.

**To Reproduce**
1. Open a score
2. Write a fermata on a barline
3. Notice error

**Expected behavior**
There should be a pause with the fermata between the last bar and the next one 

**Platform information**
 - OS: macOS Monterrey 12.6.1

**Comment by Jojo-Schmitz** [comment]
Like with a breath mark or caesura, I guess

**Comment by KaiVinter** [comment]
Exactly, but usually longer ("an indefinite amount of time") than a caesura
